sqllogictest-bin/src/main.rs

```diff
@@ -332,11 +340,14 @@ async fn run_parallel(
 
     let mut failed_case = vec![];
     let mut failed_db: HashSet<String> = HashSet::new();
+    let mut remaining_files: HashSet<String> = HashSet::from_iter(filenames.clone());
 
     let start = Instant::now();
 
     while let Some((db_name, file, res, mut buf)) = stream.next().await {
+        remaining_files.remove(&file);
```

Comment:
`filenames` (thus `remaining_files`) are normalized, while here we try to remove an original file name from it.